configuration/pih/reports/sql/covid19encounterexport.sql

```diff
@@ -6,98 +6,1328 @@
+UPDATE temp_encounter 
+SET 
+    encounter_location = ENCOUNTER_LOCATION_NAME(encounter_id);
+
+UPDATE temp_encounter 
+SET 
+    health_care_worker = OBS_VALUE_CODED_LIST(encounter_id, 'CIEL', '5619', 'fr');
+
+UPDATE temp_encounter 
+SET 
+    pregnant = OBS_VALUE_CODED_LIST(encounter_id, 'CIEL', '5272', 'fr');
+
+-- last menstruation date
+UPDATE temp_encounter te
+        LEFT JOIN
+    obs o ON o.concept_id = CONCEPT_FROM_MAPPING('CIEL', '1427')
+        AND o.person_id = te.patient_id
+        AND o.encounter_id = te.encounter_id
+        AND o.voided = 0 
+SET 
+    te.last_menstruation_date = o.value_datetime;
+
+-- estimated delivery date
+UPDATE temp_encounter te
+        LEFT JOIN
+    obs o ON o.concept_id = CONCEPT_FROM_MAPPING('CIEL', '5596')
+        AND o.person_id = te.patient_id
+        AND o.encounter_id = te.encounter_id
+        AND o.voided = 0 
+SET 
+    te.estimated_delivery_date = o.value_datetime;
+
+-- postpartum state
+UPDATE temp_encounter 
+SET 
+    postpartum_state = OBS_SINGLE_VALUE_CODED(encounter_id,
+            'CIEL',
+            '162747',
+            'CIEL',
+            '129317');
+
+-- outcome
+UPDATE temp_encounter 
+SET 
+    outcome = OBS_VALUE_CODED_LIST(encounter_id, 'CIEL', '161033', 'fr');
+
+-- date of delivery
+UPDATE temp_encounter te
+        LEFT JOIN
+    obs o ON o.concept_id = CONCEPT_FROM_MAPPING('CIEL', '5599')
+        AND o.person_id = te.patient_id
+        AND o.encounter_id = te.encounter_id
+        AND o.voided = 0 
+SET 
+    te.date_of_delivery = o.value_datetime;
+
+-- Infant
+UPDATE temp_encounter SET gestational_outcome = OBS_VALUE_CODED_LIST(
+	encounter_id,
+    'CIEL',
+    '161033',
+    'fr'
+);
+UPDATE temp_encounter SET breast_feed = OBS_VALUE_CODED_LIST(
+	encounter_id,
+    'CIEL',
+    '5632',
+    'fr'
+);
+UPDATE temp_encounter SET vaccination_up_todate = OBS_VALUE_CODED_LIST(
+	encounter_id,
+    'CIEL',
+    '5585',
+    'fr'
+);
+
+-- home medication
+UPDATE temp_encounter 
+SET 
+    home_medications = OBS_VALUE_TEXT(encounter_id, 'CIEL', '162165');
+
+UPDATE temp_encounter 
+SET 
+    allergies = OBS_VALUE_TEXT(encounter_id, 'CIEL', '162141');
+
+UPDATE temp_encounter te
+        LEFT JOIN
+    obs o ON o.concept_id = CONCEPT_FROM_MAPPING('CIEL', '1730')
+        AND o.person_id = te.patient_id
+        AND o.encounter_id = te.encounter_id
+        AND o.voided = 0 
+SET 
+    te.symptom_start_date = o.value_datetime;
+
+-- symptoms
+UPDATE temp_encounter 
+SET 
+    symptoms = OBS_VALUE_CODED_LIST(encounter_id, 'CIEL', '1727', 'fr');
+
+-- other symptoms
+UPDATE temp_encounter 
+SET 
+    other_symptoms = OBS_VALUE_TEXT(encounter_id, 'CIEL', '165996');
+
+UPDATE temp_encounter 
+SET 
+    comorbidities = OBS_VALUE_CODED_LIST(encounter_id, 'PIH', '12976', 'fr');
+
+-- comorbidities
+UPDATE temp_encounter 
+SET 
+    available_comorbidities = OBS_VALUE_CODED_LIST(encounter_id, 'CIEL', '162747', 'fr');
+
+-- other comorbidities
+UPDATE temp_encounter te
```

Comment:
We use the obs comments field?  OK.  If so, please move this into a function and then use the function.